browser.js

```diff
@@ -5,12 +5,18 @@ module.exports = function () {
     set: function (name, value, options) {
       return document.cookie = cookie.serialize(name, value, options);
     },
+
     get: function (name) {
       return cookie.parse(document.cookie)[name];
     },
-    remove: function (name) {
-      return !!(document.cookie = cookie.serialize(name, '', { expires: new Date(0) }));
+
+    remove: function (name, options) {
+      var opts = options || {};
+      opts.expires = new Date(0);
+
+      return !!(document.cookie = cookie.serialize(name, '', opts));
```

Comment:
Agreed that it's out of scope, should have mentioned that it was not new to this PR and not a blocker, just something I noticed while looking at this one :+1: